server/migrator/proto/migrator.proto

```diff
@@ -40,6 +39,24 @@ message ExportData {
   }
 }
 
+message ImportData {
+  message Req {
+    string database = 1;
+    string filename = 2;
+    map<string, string> remap_labels = 3;
+  }
+}
+
+message GetSchema {
+  message Req {
```

Comment:
one question we never got to before: can you test this with a giant schema? just like auto generate one in python or something. We don't currently know how big of a schema export we can do (has to do with grpc message size limit). We know there's 1 or 2 users out there with giant schemas, but might be ok to break, but also would be good to know what the limit is at all